### DISCUSSION TITLE
Resource files are not being saved in Production environment

### DISCUSSION THREAD

**Discussion by julianadormon** [LazZiya/XLocalizer#45] [Q&A]
Hi! 
When I run my app in my local Development environment, the resources files are saved and updated to my LocalizationResources folder and the logs say:

Save resource to file, status: 'True', key: 'Legal', value: 'Právo'

Without making any changes to the configuration options, after my app runs on the server in Production mode, the files are not being created and the logs say:

Save resource to file, status: 'False', key: 'Legal', value: 'Právo'

I cannot see how to change this so files are being saved on the server. Is this possible? Thanks so much :)

**Comment by LazZiya** [comment]
Hi,

Make sure you have the **LocalizationResources** folder with read/write permissions given to the app.

**Reply by julianadormon** [reply]
Thank you so much! I really appreciate you and this project. It's a real time-saver.

**Reply by LazZiya** [reply]
Glad to help :)

**Reply by julianadormon** [reply]
Hi @LazZiya 
I hope you are doing well.  I was hoping you could answer a question for me.
I see that the entries in the XML file created by XLocalizer look like:

  <data isActive="false">
    <key>High-Res</key>
    <value>Haute résolution</value>
    <comment>Created by XLocalizer</comment>
  </data>
  
  What does the isActive="false" mean? These were freshly added entries.
  I did change on of the values manually, but it does not seem to be used even after restarting the website in IIS.
  I wonder if the isActice=false has something to do with it?
  
  Much appreciated!

**Reply by julianadormon** [reply]
Actually I see that switching it to isActive = true and restarting IIS worked. Is there a way to automatically set these to isActive=true?

**Reply by julianadormon** [reply]
Me again. I actually see that the value is used regardless of the isActive setting. I guess there is some kind of delay.

**Reply by LazZiya** [reply]
Hi Julian, 
Everything is good, hope you also doing good:)

Currently I am in a business trip outside my country, but just for your info; this value is only for manually confirming auto translations, so you can set it to true if you have confirmed the translation manually. Or you may look fır only true values in your app, so it is up to you :)

**Reply by julianadormon** [reply]
Appreciate you! Thanks.
